src/main/webapp/js/instructorCourseDetails.es6

```diff
@@ -1,84 +1,91 @@
-'use strict';
+/* global toggleSort:false selectElementContents:false attachEventToDeleteStudentLink:false setStatusMessage:false */
+/* global BootboxWrapper:false StatusType:false */
 
-$(document).ready(function() {
-    if ($('#button_sortstudentsection').length) {
-        toggleSort($('#button_sortstudentsection'));
-    } else {
-        toggleSort($('#button_sortstudentteam'));
-    }
+function attachEventToRemindStudentsButton() {
```

Comment:
It is possible to move this back down? We can handle `no-use-before-define` in a separate set of PRs, as per Slack discussion.